doc/man7/EVP_PKEY-MLKEM.pod

```diff
@@ -0,0 +1,112 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY-ML-KEM-512,
+EVP_PKEY-ML-KEM-768,
+EVP_PKEY-ML-KEM-1024,
+EVP_KEYMGMT-ML-KEM-512,
+EVP_KEYMGMT-ML-KEM-768,
+EVP_KEYMGMT-ML-KEM-1024,
+EVP_PKEY-MLKEM
+- ML-KEM keytype and algorithm support
+
+=head1 DESCRIPTION
+
+The B<ML-KEM-512>, B<ML-KEM-768>, and B<ML-KEM-1024> keytypes are implemented in
+OpenSSL's default provider
+=for comment (TODO(ML-KEM): Add FIPS support).
```

Comment:
POD acts in paragraph mode, so you *must* have an empty line before the `=for` line as well for it to function as intended.

Reply:
thx. corrected.